server/schedulers/hot_region_config.go

```diff
@@ -82,6 +83,7 @@ type hotRegionSchedulerConfig struct {
 	MinorDecRatio          float64  `json:"minor-dec-ratio"`
 	SrcToleranceRatio      float64  `json:"src-tolerance-ratio"`
 	DstToleranceRatio      float64  `json:"dst-tolerance-ratio"`
+	PendingAmpFactor       float64  `json:"pending-amp-factor"`
```

Comment:
How to choose the right value for this config? Assume that if we encounter a hot spot problem online, what is the right way to change it? If we don't know, then this config is meanless.

Reply:
Smaller, meaning that the influence of pending will be smaller, while other solution spaces may be searched for, but maybe unstable.
Larger means that the impact of pending will be greater, while scheduling may be slower and more stable.
As for the exact number, there may not be an exact conclusion, at least not at the moment.

Reply:
What I wanna emphasize is if there is not an exact conclusion for adjusting this config, we'd better not expose it or just use a constant. Once we have a best practice for it, then make it configurable.

Reply:
use a constant